### PR TITLE
Statistics data for vaccinations (EXPOSUREAPP-7430)

### PR DESCRIPTION
**Pull request by SamuraiKek**
Adds three new statistics cards with vaccination stats:
- Persons with one vaccination dose
- Persons completely vaccinated
- Applied vaccination rates 

Currently only testable with the mock server

### CODE REVIEW

**Review by NataliaLemmerth — DISMISSED** [review]
UA approved

File: Corona-Warn-App/src/main/res/layout/home_statistics_cards_vaccinated_completely_layout.xml [hunk 1]
```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (AlexanderAlferov) [inline Corona-Warn-App/src/main/res/layout/home_statistics_cards_vaccinated_completely_layout.xml:2]: Could this wrapper layout be removed?

File: Corona-Warn-App/src/main/res/layout/home_statistics_cards_vaccinated_once_layout.xml [hunk 2]
```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
```
[the remark this answers is not in the document]
Comment (SamuraiKek) [reply Corona-Warn-App/src/main/res/layout/home_statistics_cards_vaccinated_once_layout.xml:2]: Tried it without it but if I remove it, the screen becomes invisible in the design editor. Have no idea why, so I put it back in. The rest of the statistics cards seem to have it too so at least it's somewhat consistent :D 
If you know why this happens and how to fix it, I'm happy to get rid of it.

File: Corona-Warn-App/src/main/res/layout/home_statistics_cards_vaccinated_completely_layout.xml (hunk 1 above)
Comment (SamuraiKek) [reply Corona-Warn-App/src/main/res/layout/home_statistics_cards_vaccinated_completely_layout.xml:2]: Same as the one above☝️

File: Corona-Warn-App/src/main/res/layout/home_statistics_cards_vaccinated_once_layout.xml (hunk 2 above)
[the remark this answers is not in the document]
Comment (SamuraiKek) [reply Corona-Warn-App/src/main/res/layout/home_statistics_cards_vaccinated_once_layout.xml:2]: I've found out what's causing the layout to disappear. It's this line of code `tools:showIn="@layout/home_statistics_cards_basecard_layout"`. So I guess we either leave the `layout` tags in or remove them but then also remove the `showIn' property. What do you think?

**Review by jurajkusnier — COMMENTED** [review]
Cards are not readable when font size is increased. I remember it was one of the requirements when we implement other cards.
image

**Review by jurajkusnier — APPROVED** [review]
LGTM

**Review by axelherbstreith — APPROVED** [review]
Lgtm 👍 love that you also added dedicated accessiblility strings 💯

**Review by NataliaLemmerth — APPROVED** [review]
UA approved

**Pull request merged**